Library/Homebrew/system_config.rb

```diff
@@ -112,9 +112,9 @@ def dump_verbose_config(f = $stdout)
         f.puts "Core tap: N/A"
       end
       defaults_hash = {
-        HOMEBREW_PREFIX: "/usr/local",
-        HOMEBREW_REPOSITORY: "/usr/local/Homebrew",
-        HOMEBREW_CELLAR: "/usr/local/Cellar",
+        HOMEBREW_PREFIX: Homebrew::DEFAULT_PREFIX,
+        HOMEBREW_REPOSITORY: "#{Homebrew::DEFAULT_PREFIX}/Homebrew",
```

Comment:
Does this need tweaked for Linux?

Reply:
The default repository on Linux is `/home/linuxbrew/.linuxbrew/Homebrew`, so it's good.